<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimens천es do novo escultor - Escultor 3D</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelX">
       <property name="text">
        <string>Dimens찾o X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderX">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberX">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelY">
       <property name="text">
        <string>Dimens찾o Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderY">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberY">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelZ">
       <property name="text">
        <string>Dimens찾o Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderZ">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberZ">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
